header/headertest/testing.go

```diff
@@ -111,14 +111,12 @@ func MakeCommit(
 	validators []types.PrivValidator,
 	now time.Time,
 ) (*types.Commit, error) {
-	commitSigs := make([]tmproto.CommitSig, len(validators))
-
-	for i := range validators {
+	// all sign
+	for i := 0; i < len(validators); i++ {
```

Comment:
Just curious what was the point to change `for i := range validators {` to 3-clause `for` ?

Reply:
it looks like it's like this one main 🤔 , this might be showing up as a diff since I made a change then changed it back based on @damiannolan mentioning I can use the `MakeExtendedCommit`